README.md

```diff
@@ -26,6 +26,9 @@ When you are done, deploy Sakai to Tomcat:
 mvn clean install sakai:deploy -Dmaven.tomcat.home=/path/to/your/tomcat
 ```
 
+You **must** also follow the instructions mentioned in the below link to setup mysql and other configurations(Step 9 is important).
```

Comment:
Rather than this I would give specifics so we don't need to rely on confluence. We can also just spin up an Oracle box, aka https://github.com/hilverd/vagrant-ubuntu-oracle-xe